libs/dal/src/repositories/message/message.repository.ts

```diff
@@ -22,10 +23,9 @@ export class MessageRepository extends BaseRepository<MessageDBModel, MessageEnt
     environmentId: string,
     subscriberId: string,
     channel: ChannelTypeEnum,
-    query: { feedId?: string[]; seen?: boolean; read?: boolean } = {},
-    options: { limit: number; skip?: number } = { limit: 10 }
```

Comment:
Removed 'options' because it was not used internally in the method in order to create the query.